plugins/outputs/stomp/stomp_test.go

```diff
@@ -0,0 +1,58 @@
+package stomp
+
+import (
+	"testing"
+	"time"
+
+	"github.com/go-stomp/stomp"
+	"github.com/stretchr/testify/require"
+
+	"github.com/influxdata/telegraf/plugins/serializers"
+	"github.com/influxdata/telegraf/testutil"
+)
+
+// TestiConnectAndWrite ...
+func TestConnectAndWrite(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration test in short mode")
+	}
+
+	url := testutil.GetLocalHost() + ":61613"
```

Comment:
Can you maybe use a random port here to avoid conflicts with multiple tests running at the same time...